examples/simple-service-poc/simple_service/simple_service.Dockerfile

```diff
@@ -1,9 +1,9 @@
 FROM python:3.8-slim
 VOLUME /golem/in /golem/out
+RUN pip install numpy matplotlib python-daemon
```

Comment:
I moved this up, so docker can reuse this layer from the cache when working on the python scripts and rebuilding the image frequently.